projects/lib-shared-modules/src/lib/services/form/form.service.ts

```diff
@@ -108,6 +108,23 @@ export class FormService {
     }
   }
 
+  getCertificateForm(){
+    return new Promise((resolve, reject) => {
```

Comment:
why return as promise ? 
why this function first of all ?
there is a common function to use to get forms use that.